### PR TITLE
STYLE upgrade black formatter

### PR DESCRIPTION
**Pull request by MarcoGorelli**
gonna have to do this eventually...

I'm trying to tell if it changed anything other than just deleting newlines, any `git` masters know how to do that?

### CODE REVIEW

File: pandas/_libs/hashtable.pyi
Comment (rhshadrach) [inline pandas/_libs/hashtable.pyi:194]: Here

File: pandas/conftest.py
Comment (rhshadrach) [inline pandas/conftest.py:171]: Here

File: pandas/core/algorithms.py
Comment (rhshadrach) [inline pandas/core/algorithms.py:1698]: Here

File: pandas/core/apply.py
Comment (rhshadrach) [inline pandas/core/apply.py:940]: Here

File: pandas/core/dtypes/concat.py
Comment (rhshadrach) [inline pandas/core/dtypes/concat.py:51]: Here

File: pandas/core/reshape/encoding.py
Comment (rhshadrach) [inline pandas/core/reshape/encoding.py:201]: Here

File: pandas/core/series.py
Comment (rhshadrach) [inline pandas/core/series.py:238]: Here

File: pandas/io/formats/style_render.py
Comment (rhshadrach) [inline pandas/io/formats/style_render.py:2154]: Here

File: pandas/io/formats/style_render.py
Comment (rhshadrach) [inline pandas/io/formats/style_render.py:2297]: Here

File: pandas/plotting/_matplotlib/core.py
Comment (rhshadrach) [inline pandas/plotting/_matplotlib/core.py:718]: Here

File: pandas/tests/arithmetic/conftest.py
Comment (rhshadrach) [inline pandas/tests/arithmetic/conftest.py:23]: Here

File: pandas/tests/arrays/categorical/test_indexing.py
Comment (rhshadrach) [inline pandas/tests/arrays/categorical/test_indexing.py:367]: Here

File: pandas/tests/generic/test_duplicate_labels.py
Comment (rhshadrach) [inline pandas/tests/generic/test_duplicate_labels.py:184]: Here

File: pandas/tests/generic/test_label_or_level_utils.py
Comment (rhshadrach) [inline pandas/tests/generic/test_label_or_level_utils.py:128]: Here

File: pandas/tests/indexes/datetimes/test_setops.py
Comment (rhshadrach) [inline pandas/tests/indexes/datetimes/test_setops.py:228]: Here

File: pandas/tests/indexes/datetimes/test_setops.py
Comment (rhshadrach) [inline pandas/tests/indexes/datetimes/test_setops.py:259]: Here

File: pandas/tests/indexes/period/test_setops.py
Comment (rhshadrach) [inline pandas/tests/indexes/period/test_setops.py:195]: Here

File: pandas/tests/indexes/period/test_setops.py
Comment (rhshadrach) [inline pandas/tests/indexes/period/test_setops.py:229]: Here

File: pandas/tests/indexes/timedeltas/test_scalar_compat.py
Comment (rhshadrach) [inline pandas/tests/indexes/timedeltas/test_scalar_compat.py:106]: Here

File: pandas/tests/window/test_expanding.py
Comment (rhshadrach) [inline pandas/tests/window/test_expanding.py:188]: Here

File: pandas/tests/window/test_expanding.py
Comment (rhshadrach) [inline pandas/tests/window/test_expanding.py:207]: Here

File: pandas/tests/window/test_rolling.py
Comment (rhshadrach) [inline pandas/tests/window/test_rolling.py:799]: Here

File: pandas/tests/window/test_rolling.py
Comment (rhshadrach) [inline pandas/tests/window/test_rolling.py:895]: Here

File: pandas/tests/window/test_rolling.py
Comment (rhshadrach) [inline pandas/tests/window/test_rolling.py:945]: Here

File: setup.py
Comment (rhshadrach) [inline setup.py:396]: Here

**Review by rhshadrach — COMMENTED** [review]
Highlighting changes other than removed new lines and the change to black versions

The added newlines are interesting; not sure why it's doing that.

File: setup.py
Comment (MarcoGorelli) [reply setup.py:396]: thanks! did you do this manually or do you have some magical `git diff` option? 🧙

**Comment by MarcoGorelli** [comment]
thanks @rhshadrach for going through the changes - I think they look fine

@lithomas1 is it OK to go ahead with this, or does it need to wait til after the release?

File: setup.py
Comment (rhshadrach) [reply setup.py:396]: Ah, just manual.

**Comment by lithomas1** [comment]
+1 on landing, let's do this before the branch cut.

**Comment by MarcoGorelli** [comment]
cool, I've updated - let's merge on green-ish then?

**Review by lithomas1 — APPROVED** [review]

**Comment by mroeschke** [comment]
Sorry I just merged a few PRs since this got updated.

Since it's a style change that could impact all PRs, lets rebase once more (I won't merge any other PRs) and get this in

**Comment by MarcoGorelli** [comment]
I just tried fetching and merging upstream/main locally, and then did `pre-commit run --all-files`, and everything passed

So I think we should be safe to just merge directly

**Comment by mroeschke** [comment]
Ah nice thanks for checking.

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @MarcoGorelli

**Comment by MarcoGorelli** [comment]
thank you

> So I think we should be safe to just merge directly

If this doesn't work I accept full blame

**Pull request merged**
